setup.py

```diff
@@ -8,9 +8,24 @@
 """
 
 import re
+import sys
 
 from setuptools import setup, find_packages
 
+def _get_requirements(path):
+    try:
+        with open(path) as f:
+            packages = f.read().splitlines()
+    except (IOError, OSError) as ex:
+        raise RuntimeError("Can't open file with requirements: %s", repr(ex))
+    return [p.strip() for p in packages if not re.match(r"^\s*#", p)]
+
+def _install_requirements():
+    requirements = _get_requirements('requirements.txt')
+    if sys.version_info[0] >= 3:
+        requirements += _get_requirements('requirements-py3.txt')
```

Comment:
Looks like `requirements-py3.txt` is an empty file. Place holder for the future?